### ISSUE TITLE
apker / db problems at boot

### ISSUE THREAD

**Issue by reyammer** [CTFd/CTFd#1656]
when apker starts and db is not there yet, apker keeps crashing with connection refused. It should just wait

**Comment by ColdHeat** [comment]
Is this issue for the right repo? I'm not sure what apker is?

**Comment by reyammer** [comment]
LOL. sorry, hub CLI suddenly started pointing to your repo vs. my private fork. Never mind :-)

**Comment by ColdHeat** [comment]
Just curious, is apker this project? https://github.com/hypertasks/apker
Seems like an interesting alternative to terraform?

**Comment by reyammer** [comment]
nope, but that looks pretty nice! apker is one my components to run https://challs.reyammer.io/, it's basically ctfd + APK (Android apps) analysis system, hooked up with an Android emulator to actually run the apps + interface to allow users to submit APKs / show logs (so that they can exfiltrate flags out of them). Not ready yet for open source, but ofc happy to share code :)

About the "hub" bug, that's interesting/scary: I have "origin" remote pointing to my private fork, and hub always pointed there. A couple days ago I started porting my ctfd fork (~2 years old...) to ctfd 3.1, and I added an "upstream" remote pointing to your public repo... and somehow "hub" silently decided to use that as main reference. Scary stuff.
